src/procedures/FinalizeSto.ts

```diff
@@ -75,12 +75,7 @@ export class FinalizeSto extends Procedure<FinalizeStoProcedureArgs> {
       });
     }
 
-    function throwStoModuleError() {
-      throw new PolymathError({
-        code: ErrorCode.ProcedureValidationError,
-        message: `STO ${stoAddress} is either archived or hasn't been launched`,
-      });
-    }
+    const stoModuleErrorMessage = `STO ${stoAddress} is either archived or hasn't been launched`;
```

Comment:
What I meant was to assign the entire error to a variable (i.e. const `stoModuleError = new PolymathError({ ... })`)